Description: don't fail on circular dependencies
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-11-09

--- a/rollup.config.ts
+++ b/rollup.config.ts
@@ -29,7 +29,7 @@
 			'Please keep in mind that the browser build may never have external dependencies!'
 	);
 	// eslint-disable-next-line unicorn/error-message
-	throw Object.assign(new Error(), warning);
+	//throw Object.assign(new Error(), warning);
 };
 
 const treeshake = {
