Description: fix for node-signal-exit >= 4.1
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2023-08-15

--- a/cli/run/watch-cli.ts
+++ b/cli/run/watch-cli.ts
@@ -28,6 +28,7 @@
 	const configFile = command.config ? await getConfigPath(command.config) : null;
 	const runWatchHook = createWatchHooks(command);
 
+// @ts-ignore
 	onExit(close);
 	process.on('uncaughtException', closeWithError);
 	if (!process.stdin.isTTY) {
