Description: fix typescript for node-magic-string >= 0.30
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2023-08-15

--- a/build-plugins/add-cli-entry.ts
+++ b/build-plugins/add-cli-entry.ts
@@ -16,6 +16,7 @@
 			});
 		},
 		name: 'add-cli-entry',
+// @ts-ignore
 		renderChunk(code, chunkInfo) {
 			if (chunkInfo.fileName === CLI_CHUNK) {
 				const magicString = new MagicString(code);
--- a/src/Chunk.ts
+++ b/src/Chunk.ts
@@ -347,6 +347,7 @@
 				])
 			),
 			imports: renderedChunkInfo.imports.map(finalize),
+// @ts-ignore
 			map,
 			preliminaryFileName,
 			referencedFiles: renderedChunkInfo.referencedFiles.map(finalize)
--- a/src/utils/transform.ts
+++ b/src/utils/transform.ts
@@ -129,6 +129,7 @@
 						error_.hook = 'transform';
 						return pluginContext.error(error_);
 					},
+// @ts-ignore
 					getCombinedSourcemap() {
 						const combinedMap = collapseSourcemap(
 							id,
