Description: ignore failure on dynamic import
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-04-25

--- a/build-plugins/esm-dynamic-import.ts
+++ b/build-plugins/esm-dynamic-import.ts
@@ -1,7 +1,7 @@
 import type { Plugin } from 'rollup';
 
 export default function esmDynamicImport(): Plugin {
-	let importFound = false;
+	let importFound = true;
 	return {
 		generateBundle() {
 			if (!importFound) {
